docs/rst/formalia/titlepage.rst

```diff
@@ -11,5 +11,46 @@
     :alt: eProsima
     :target: http://www.eprosima.com/
 
-Description
-^^^^^^^^^^^
+*eProsima Fast DDS QoS Profiles Manager* is a tool suite for the generation of `Fast DDS <https://fast-dds.docs.eprosima.com/en/latest/>`_ `XML configuration files <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html>`_.
+The suite provides both a Graphical User Interface (GUI) and a Command Line Interface (CLI).
+
+Graphical User Interface
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+*eProsima Fast DDS QoS Profiles Manager* provides a GUI to facilitate users the creation and modification of XML configuration files in an easy way.
+With an easy to use interface which allows to navigate between the configuration menus, the corresponding features can be configured and the XML to be deployed can be validated and created.
+Thus, the user can be agnostic of the specific XML format followed by *eProsima Fast DDS*.
+More information can be found in the :ref:`fastdds_qos_profiles_manager_gui` section.
```

Comment:
It looks weird if the same name matches with the current section, but as long as the GUI official name does not have the "GUI" on it, I think it the clarification is useful. 
```suggestion
More information can be found in the :ref:`fastdds_qos_profiles_manager_gui` GUI section.
```

Reply:
Let us wait for @EduPonz feedback concerning the official names.